BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoProceedView.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  SkontoProceedView.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+import GiniCaptureSDK
+
+class SkontoProceedView: UIView {
+    private lazy var payButton: MultilineTitleButton = {
+        let button = MultilineTitleButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.configure(with: configuration.primaryButtonConfiguration)
+        button.titleLabel?.font = configuration.textStyleFonts[.bodyBold]
+        let buttonTitle = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.paybutton.title",
+                                                                   comment: "Proceed")
+        button.accessibilityValue = buttonTitle
+        button.setTitle(buttonTitle, for: .normal)
+        return button
+    }()
+
+    private lazy var totalLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.adjustsFontForContentSizeCategory = true
+        label.font = configuration.textStyleFonts[.body]
+        label.textColor = .giniColorScheme().text.primary.uiColor()
+        let labelTitle = NSLocalizedStringPreferredGiniBankFormat("ginibank.digitalinvoice.lineitem.totalpricetitle",
```

Comment:
[suggestion] We should use a dedicated localizabale string for each new text for skonto.